docs/docs/cmd/spo/homesite/homesite-remove.mdx

```diff
@@ -31,12 +34,18 @@ To use this command you must be either **SharePoint Administrator** or **Global
 
 ## Examples
 
-Removes the current Home Site without confirmation.
+Removes the first Home Site without confirmation.
 
 ```sh
 m365 spo homesite remove --force
 ```
```

Comment:
Since we don't want people to use this approach anymore, let's just remove it as example.